<launch>
	<!--

  	<node name="ds4drv_script_call" pkg="ps4_ros" type="run_ds4drv.sh" output="screen" />


	<node respawn="true" pkg="joy" type="joy_node" name="ps4_joy" >
    	<param name="dev" type="string" value="/dev/input/js0" />
    	<param name="deadzone" value="0.12" />
 	</node>

-->
	<!-- PS4 ROS NODE -->
	<node pkg="ps4_ros" type="ps4_ros" name="ps4" output="screen">
	  	<param name="scale_linear" value="3.0"/>
	  	<param name="scale_angular" value="3"/>
	  	<param name="pub_topic" value="/vel_cmd_joystick"/>
	</node>

</launch>


